webapp/tests/forms/steps/lotse_multistep_flow_steps/test_steuerminderungen_steps.py

```diff
@@ -205,22 +205,42 @@ def test_do_not_skip_if_separated(self):
         assert redirection_info[0] is None
         assert redirection_info[1] is None
 
-    def test_skip_if_widowed_zusammenveranlagung(self):
+    def test_do_not_skip_if_widowed_longer(self):
+        separated_data = {'steuerminderung': 'yes', 'stmind_handwerker_summe': 14,
+                          'familienstand': 'widowed',
+                          'familienstand_date': datetime.date(1990, 1, 2)}
```

Comment:
I was just wondering if it might make sense to set that data to the last day of the last year out of the Veranlagungszeitraum?

Reply:
I just saw that you did something like that already (e.g. l. 228)?